app/Http/Controllers/Admin/UserController.php

```diff
@@ -82,9 +124,34 @@ public function edit($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(UserRequest $request, $id)
     {
-        //
+        $user = $this->userRepo->find($id);
+        if ($user) {
+            $data = [
+                'name' => $request->name,
+                'address' => $request->address,
+                'email' => $request->email,
+                'phone' => $request->phone,
+                'role_id' => $request->role,
+                'status' => 1,
+            ];
+            if ($request->hasFile('image')) {
+                $file = $request->file('image');
+                $name = time() . "_" . $file->getClientOriginalName();
```

Comment:
đồng nhất các ngoặc nhé ' ' hoặc cùng " " 
```suggestion
                $name = time() . '_' . $file->getClientOriginalName();
```